frontend/src/metabase/query_builder/components/ResponsiveParametersList.styled.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from "@emotion/styled";
+import Button from "metabase/core/components/Button";
+import SyncedParametersList from "metabase/parameters/components/SyncedParametersList";
+
+import { color } from "metabase/lib/colors";
+
+export const FilterButton = styled(Button)`
+  color: ${color("brand")};
+  margin: 0.5rem;
+`;
+
+export const ResponsiveParametersListRoot = styled.div`
+  width: 100%;
+`;
+
+interface ParametersListContainerProps {
+  isSmallScreen: boolean;
+  mobileShow: boolean;
+}
+
+export const ParametersListContainer = styled.div<ParametersListContainerProps>`
+  background-color: ${color("bg-light")};
+
+  ${({ isSmallScreen, mobileShow }) =>
+    isSmallScreen &&
+    `
```

Comment:
Pro-tip. Use `css` from `@emotion/react` to make syntax highlighting works (if you use VS code) 😄 

#### without `css`
![image](https://user-images.githubusercontent.com/1937582/183598495-c5df394e-afe2-4f66-844a-d58faa0eb7e6.png)

#### with `css`
![image](https://user-images.githubusercontent.com/1937582/183598447-0505d6ea-523d-4794-ba22-59d1e714bb4a.png)

Reply:
Huh! @npfitz thanks for pointing that out. I think I have [vscode-styled-components](https://marketplace.visualstudio.com/items?itemName=diegolincoln.vscode-styled-components#:~:text=The%20styled%2Dcomponents%20extension%20adds,linter%20and%20other%20language%20features.) plugin installed. I just tested that disabling that plugin removes the syntax highlighting for me. Cheers!